marvin_strings.json

```diff
@@ -239,7 +239,7 @@
         "error": "Något gick snett, jag har inte en susning om någon fyller år eller inte."
     },
     "nameday": {
-        "url": "http://www.dagensnamn.nu",
+        "url": "http://api.dryg.net/dagar/v2.1/{year}/{month}/{day}",
```

Comment:
should this really be in strings.json? It makes the code harder to read when you are trying to figure out what the .format() is formatting.